### PR TITLE
Bumping varfish-annotator-cli (#168)

### PR DESCRIPTION
**Pull request by holtgrewe**
Should only be merged when this is live on the BIH VarFish instance.

Must be merged after #165.

### CODE REVIEW

File: snappy_wrappers/tools/vcf_merge_exome_cnvs.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
 import attr
 import logzero
 from logzero import logger
-import pandas as pd
```
Comment (eudesbarbosa) [inline snappy_wrappers/tools/vcf_merge_exome_cnvs.py:14]: Issue with import order in CI,`isort`.

**Review by eudesbarbosa — COMMENTED** [review]
Looks good, just small CI issue.

**Comment by holtgrewe** [comment]
CI needed a refurbishing. I now removed Codacy in favour of flake8. I also pinned the black etc. versions to something recent and added Python 3.10.

File: snappy_pipeline/apps/snappy_pull_sheet.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def run(args):
     study_top = study_infos["top_header"]
     n_source = study_top[0]["colspan"]
     n_extraction = study_top[1]["colspan"]
-    n_sample = study_top[2]["colspan"]
+    # n_sample = study_top[2]["colspan"]
```
Comment (eudesbarbosa) [inline snappy_pipeline/apps/snappy_pull_sheet.py:66]: Consider removing commented code.

Comment (holtgrewe) [reply snappy_pipeline/apps/snappy_pull_sheet.py:66]: Good point, done.

File: snappy_wrappers/tools/vcf_merge_exome_cnvs.py [hunk 3]
```diff
@@ -142,10 +141,10 @@ class CnvCluster:
 
 
 def merge_headers(headers: typing.Iterable[vcfpy.Header]) -> vcfpy.Header:
-    """"Merge Headers for output."""
+    """ "Merge Headers for output."""
```
Comment (eudesbarbosa) [inline snappy_wrappers/tools/vcf_merge_exome_cnvs.py:144]: Extra quote

File: snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/coverage/wrapper.py [hunk 4]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", ".."))
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/coverage/wrapper.py:10]: No need for hack solution any more?

File: snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/merge_cohort_vcfs/wrapper.py [hunk 5]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", ".."))
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/merge_cohort_vcfs/wrapper.py:10]: No need for hack solution any more?

File: snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/merge_neighbors/wrapper.py [hunk 6]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", ".."))
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/cnvetti/targeted_germline_hom/merge_neighbors/wrapper.py:10]: No need for hack solution any more?

File: snappy_wrappers/wrappers/eb_filter_par/parallel_eb_filter.py [hunk 7]
```diff
@@ -12,13 +18,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/eb_filter_par/parallel_eb_filter.py:19]: No need for hack solution any more?

File: snappy_wrappers/wrappers/epitoper_par/wrapper.py [hunk 8]
```diff
@@ -9,13 +16,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/epitoper_par/wrapper.py:17]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gatk_hc_par/parallel_gatk_hc.py [hunk 9]
```diff
@@ -4,20 +4,19 @@
...
+    gib_to_string,
+    hours,
+)
+from snappy_wrappers.resource_usage import ResourceUsage
 import os
 import sys
 
 # A hack is required for being able to import snappy_wrappers modules when in development mode.
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gatk_hc_par/parallel_gatk_hc.py:18]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gatk_read_backed_phasing_par/parallel_read_backed_phasing.py [hunk 10]
```diff
@@ -13,12 +19,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gatk_read_backed_phasing_par/parallel_read_backed_phasing.py:20]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gatk_ug_par/parallel_gatk_ug.py [hunk 11]
```diff
@@ -12,13 +18,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gatk_ug_par/parallel_gatk_ug.py:19]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gcnv/annotate_gc/wrapper.py [hunk 12]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gcnv/annotate_gc/wrapper.py:11]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gcnv/filter_intervals/wrapper.py [hunk 13]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gcnv/filter_intervals/wrapper.py:11]: No need for hack solution any more?

File: snappy_wrappers/wrappers/gcnv/merge_cohort_vcfs/wrapper.py [hunk 14]
```diff
@@ -9,7 +10,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/gcnv/merge_cohort_vcfs/wrapper.py:11]: No need for hack solution any more?

File: snappy_wrappers/wrappers/jannovar_par/annotate_somatic_vcf/parallel_annotate_somatic_vcf.py [hunk 15]
```diff
@@ -12,13 +18,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/jannovar_par/annotate_somatic_vcf/parallel_annotate_somatic_vcf.py:19]: No need for hack solution any more?

File: snappy_wrappers/wrappers/jannovar_par/annotate_vcf/parallel_annotate_vcf.py [hunk 16]
```diff
@@ -12,13 +18,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/jannovar_par/annotate_vcf/parallel_annotate_vcf.py:19]: No need for hack solution any more?

File: snappy_wrappers/wrappers/variant_filtration/filter_denovo/wrapper.py [hunk 17]
```diff
@@ -13,9 +15,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/variant_filtration/filter_denovo/wrapper.py:16]: No need for hack solution any more?

File: snappy_wrappers/wrappers/xhmm/gatk_cov/wrapper.py [hunk 18]
```diff
@@ -13,7 +14,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/xhmm/gatk_cov/wrapper.py:15]: No need for hack solution any more?

File: snappy_wrappers/wrappers/xhmm/gatk_ref_stats/wrapper.py [hunk 19]
```diff
@@ -13,7 +14,6 @@
 base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
 sys.path.insert(0, base_dir)
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/xhmm/gatk_ref_stats/wrapper.py:15]: No need for hack solution any more?

**Review by eudesbarbosa — COMMENTED** [review]
Some leftover from previous import structure is remaining...
```
# A hack is required for being able to import snappy_wrappers modules when in development mode.
# TODO: is there a more elegant way?
base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
sys.path.insert(0, base_dir)
```

File: snappy_wrappers/tools/vcf_merge_exome_cnvs.py (hunk 3 above)
Comment (holtgrewe) [reply snappy_wrappers/tools/vcf_merge_exome_cnvs.py:144]: fixed

**Review by eudesbarbosa — APPROVED** [review]
Looks good, thank you for the changes.

**Pull request merged**